### DISCUSSION TITLE
can react hook form work for react 15 as i am working for SPFx for onprem solutions which does not support 16 as yet

### DISCUSSION THREAD

**Discussion by Shahid-Roofi-Khan** [react-hook-form/react-hook-form#3509] [Q&A] [answered]
can react hook form work for react 15 as i am working for SPFx for onprem solutions which does not support 16 as yet

so i need to decide between formik and react hook form.

From the title it seems it relies on react hooks which are only possible in React 16.8

Any insights please?

**Comment by bluebill1049** [comment, marked as answer]
I only work after 16.8 (hook support)
